cypress/fixtures/README.md

```diff
@@ -5,3 +5,26 @@ Fixtures are designed to load a fixed set of data located in a file.
 ## Required data need to be updated before running the feature files
 ### Advertising: 
  - APPBUNDLEID - It contains the corresponding app ID and should be updated in `configModule/testData/modules/advertising.json` for content validation.
```

Comment:
We should not add anything related to configModule inside FCS . If required, that can be added in configModule readme